esp-hal-common/src/dma/mod.rs

```diff
@@ -20,7 +20,7 @@ pub enum DmaError {
 }
 
 /// DMA Priorities
-#[cfg(any(esp32c3, esp32s3))]
+#[cfg(any(esp32c2, esp32c3, esp32s3))]
 #[derive(Clone, Copy)]
 pub enum DmaPriority {
```

Comment:
Curious where these values came from?

In the ESP32-C2's ESP32-C3's TRMs it says:

> To ensure timely response to peripherals running at a high speed with low latency (such as SPI), the GDMA controller implements a fixed-priority channel arbitration scheme. That is to say, each channel can be assigned a priority from 0 ~ 9.

Reply:
Oh yes ... I have no idea what I thought here. Mind changing it in your PR?

Reply:
Not at all, just wanted to make sure I wasn't missing something first haha